bin/copyRawDataToTmp.sh

```diff
@@ -47,21 +47,45 @@ fi
 function rsyncNGSRuns() {
 	local _samplesheet
 	_samplesheet="${1}"
-
+	log4Bash 'TRACE' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Working on ${_samplesheet}"
 	while read -r line
 	do
 		## line is ${filePrefix/${filePrefix}_1.fq.gz
 		##
-		rsync -rlptDvc --relative --rsync-path="sudo -u ${group}-ateambot rsync" "${WORKING_DIR}/./rawdata/ngs/${line}"* "${DESTINATION_DIAGNOSTICS_CLUSTER}:${TMP_ROOT_DIR}/" \
-		|| {
-			log4Bash 'ERROR' "${LINENO}" "${FUNCNAME:-main}" '0' "Failed to rsync ${line}"
-			log4Bash 'ERROR' "${LINENO}" "${FUNCNAME:-main}" '0' "    from ${WORKING_DIR}/./rawdata/ngs/"
-			log4Bash 'ERROR' "${LINENO}" "${FUNCNAME:-main}" '0' "    to ${DESTINATION_DIAGNOSTICS_CLUSTER}:${TMP_ROOT_DIR}/rawdata/ngs/"
+		log4Bash 'TRACE' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Looping through all prm mounts"
+		copied="no"
+		for prm in "${ALL_PRM[@]}"
+		do
+			log4Bash 'TRACE' "${LINENO}" "${FUNCNAME[0]:-main}" '0' "Checking for ${line} on ${prm} and if it still needs to be processed"
+			if [[ -f "/groups/${group}/${prm}/rawdata/ngs/${line}" && "${copied}" == "no" ]]
```

Comment:
- The `-f` test will only work for files and hence fail if we want to transfer a dir or symlink in the future. Why not test for whether the path exists. E.g. 
   ```
   if [[ -e "/groups/${group}/${prm}/rawdata/ngs/${line}" && "${copied}" == "no" ]]
   ```
- If `${line}` would also contain `rawdata/ngs` then this script could also work for other pipelines like the ones for processing array data. Now `rawdata/ngs/` is hard coded in front of the `${line}`making it only work for one type of data.